nanoFramework.Device.Bluetooth/BluetoothLEDevice.cs

```diff
@@ -270,11 +270,8 @@ internal set
                 BluetoothConnectionStatus currentStatus = _connectionStatus;
                 _connectionStatus = value;
 
-                if (currentStatus != value)
-                {
-                    // Fire event on change
-                    ConnectionStatusChanged?.Invoke(this, EventArgs.Empty);
-                }
+                // Fire event on change
+                ConnectionStatusChanged?.Invoke(this, EventArgs.Empty)
```

Comment:
I'm wondering if it would just be better to handle the first case or better handle internally the changes. Because that may raise a lot of needed events. Any thoughts on this? And yes, I know you've tested it.

Reply:
The event is coming from native code, so I am not sure what else to do.

Reply:
@AdrianSoundy,  Please Review the change.  Thank you.